README.md

```diff
@@ -105,8 +105,8 @@ If you're having issues with the packaged executable or using this tool in gener
 | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
 | Arabic (العربية)                            | ar    | [Tayeb-Ali](https://github.com/tayeb-ali)     | UI                                   |
 | Brazilian Portuguese (Português Brasileiro) | pt_br | [marleyas](https://github.com/marleyas)       | UI (partial)                         |
-| Chinese Simplified (简体中文)               | zh    | [jiangzhe11](https://github.com/jiangzhe11)   | UI and [README](./README-Chinese.md) |
-| Chinese Traditional (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)        | UI                                   |
+| Chinese Simplified  (简体中文)               | zh    | [jiangzhe11](https://github.com/jiangzhe11)   | UI and [README](./README-Chinese.md) |
+| Chinese Traditional  (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)        | UI                                   |
```

Comment:
Has this been done using an auto-formatter? These changes don't show up when I auto format using prettier in vscode

Reply:
It was auto-formatter, nothing important